src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-configs.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "General OpenTelemetry collector configurations"
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: Here are some general configurations for your collector.
+---
+
+A variety of general configurations are available for your collector setup.
+
+### Health Check [#health-check]
```

Comment:
Should "Check" be "check"?

Reply:
@barbnewrelic, good catch! I fixed this!